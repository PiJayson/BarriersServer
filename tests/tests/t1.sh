#!/bin/sh

. common

$BIN/series.x 2 5
$BIN/series.x 3 5
$BIN/series.x 4 5

