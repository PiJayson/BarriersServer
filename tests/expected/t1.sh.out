Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 2 ... result 0
Destroying barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 3 ... result 0
Destroying barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Calling wait, barrier id: 19
Waiting done, barrier id: 19 ... result 0
Init barrier id: 19 width: 4 ... result 0
Destroying barrier id: 19 ... result 0
